core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1149,6 +1149,19 @@
 
   public static final String USE_KETTLE_DEFAULT = "false";
 
+  /**
+   * Sorts the data in batches and writes the batch data to store with index file.
+   */
+  public static final String LOAD_USE_BATCH_SORT = "carbon.load.use.batch.sort";
+
+  public static final String LOAD_USE_BATCH_SORT_DEFAULT = "true";
+
+  /**
+   * Size of batch data to keep in memory, as a thumb rule it supposed
+   * to be less than 45% of sort.inmemory.size.inmb otherwise it may spill intermediate data to disk
+   */
+  public static final String LOAD_BATCH_SORT_SIZE_INMB = "carbon.load.batch.sort.size.inmb";
```

Comment:
I think it is better to move it up nearby  where `IN_MEMORY_FOR_SORT_DATA_IN_MB` is